src/helpers/classNames/classNames.js

```diff
@@ -4,5 +4,5 @@ export const classNames = (...classes) => {
     && className.trim().length > 0,
   );
 
-  return filteredClassNames.length > 0 ? filteredClassNames.join(' ') : undefined;
+  return filteredClassNames.length > 0 ? filteredClassNames.join(' ') : '';
```

Comment:
Why empty string and not `undefined`? We want `undefined` to completely omit parameter from the DOM. Empty strings will polute snapshots and DOM, thus this would have been marked as BC.

Reply:
You are right. I thought we agree o this on our call to always return the same type for use with TS.

I will revert it and a comment.

Reply:
I am not aware of it. But if we did, I am sorry for that.

Reply:
And quick note, you changed test names as well.

Reply:
done